index.html

```diff
@@ -290,7 +291,42 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
-
+    <footer>
+      <form
+        action="https://formspree.io/f/xdoyydwq"
+        method="POST"
+        class="form-container"
+      >
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <div class="inputs">
+          <div class="group-input">
+            <div class="fname">
+   
+              <input type="text" placeholder="First Name" />
+            </div>
+            <div class="lname">
+          
+              <input type="text" placeholder="Last Name"/>
```

Comment:
- [ ] The first name and last name inputs are supposed to have a `maxlength="30"` attribute according to the project requirements. Let's add this attribute since it is a major requirement and it is always good to go through the major requirements to avoid such mistakes.